### DISCUSSION TITLE
Whether to support OpenKruise's CloneSet, Advanced StatefulSet and Advanced DaemonSet and other resources

### DISCUSSION THREAD

**Discussion by kissycn** [kubevela/kubevela#5832] [Q&A] [answered]
Whether to support OpenKruise's CloneSet, Advanced StatefulSet and Advanced DaemonSet and other resources
I see only cloneset in the support files of the community. 
Do other resources like Advanced StatefulSet and Advanced DaemonSet have support plans?

**Comment by Somefive** [comment, marked as answer]
By far no plan. But you can raise an issue and see if there is any community member want to contribute to that. Or you can go on with yourself by adding more definitions to the open kruise addon.

**Comment by kissycn** [comment]
ok i'm interested
